docs/src/main/sphinx/release/release-396.md

```diff
@@ -0,0 +1,51 @@
+# Release 396 (14 Sep 2022)
+
+## General
+
+* Improve performance of queries that process string data types. ({issue}`12798`)
+* Fix failure when querying a view which uses a table function.  ({issue}`13944`)
+
+## BigQuery connector
+
+* Add support for [column comments and table comments](/sql/comment). ({issue}`13882`)
+
+## ClickHouse connector
+
+* Allow pushdown of larger predicates by default. ({issue}`14029`)
+* Extend the range of possible values of the ClickHouse `Date` and `DateTime`
+  types. ({issue}`11116`)
+
+## Delta Lake connector
+
+* Improve optimized Parquet writer performance for
```

Comment:
If I recall correctly, there's no other writer for Parquet in the Delta Lake connector. This should be rephrased as:

> Improve performance when writing structural data types to Parquet files.

Reply:
Delta Lake connector has both the parquet writers, iceberg doesn't